### ISSUE TITLE
【Forge/1.19】Unknown Crash

### ISSUE THREAD

**Issue by AmyRose1288** [Lizard-Of-Oz/Inventorio#139]
java.lang.reflect.InvocationTargetException: null

Forge 41.1.0 Minecraft 1.19 No Optifine Installed

[latest.log](https://github.com/Lizard-Of-Oz/Inventorio/files/9239058/latest.log)
[crash-2022-08-02_12.33.07-fml.txt](https://github.com/Lizard-Of-Oz/Inventorio/files/9239059/crash-2022-08-02_12.33.07-fml.txt)

**Comment by Lizard-Of-Oz** [comment]
Yeah, this is gonna be a common sight in issues trackers of many Forge mods:

> **Warning: Forge for 1.19.0 has introduced API-breaking changes in `41.1.x`.**
> Inventorio 1.8.2 version is for `1.19.1` and `1.19.0 with Forge 41.1.x`
> For `1.19.0 with Forge 41.0.x` use Inventorio 1.8.0

You need to either download the latest version (released an hour ago) or downgrade Forge.
